.devcontainer/docker-compose.yml

```diff
@@ -10,3 +10,4 @@ services:
       - ..:/workspaces/wowless
     tty: true
     working_dir: /workspaces/wowless
+    user: vscode
```

Comment:
Out of curiosity, what does this achieve?

Reply:
sets the default user when executing commands via `docker-compose exec` (as opposed to root)